pkgs/development/tools/misc/pinact/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, buildGo120Module
+, testers
+, pinact
+}:
+
+buildGo120Module rec {
+  pname = "pinact";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "suzuki-shunsuke";
+    repo = "pinact";
+    rev = "v${version}";
+    hash = "sha256-OQo21RHk0c+eARKrA2qB4NAWWanb94DOZm4b9lqDz8o=";
+  };
+
+  vendorHash = "sha256-g7rdIE+w/pn70i8fOmAo/QGjpla3AUWm7a9MOhNmrgE=";
+
+  doCheck = false;
+
+  passthru.tests.version = testers.testVersion {
+    package = pinact;
+    command = "pinact --version";
+    version = src.rev;
+  };
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version} -X main.commit=${src.rev}"
+  ];
+
+  meta = with lib; {
+    description = "Pin GitHub Actions versions";
+    homepage = "https://github.com/suzuki-shunsuke/pinact";
+    changelog = "https://github.com/suzuki-shunsuke/pinact/releases/tag/${src.rev}";
+    license = licenses.mit;
+    maintainers = [ ];
+    mainProgram = "pinact";
```

Comment:
The package should probably have a maintainer.
You can add yourself to [maintainers/maintainers-list.nix](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix).

Reply:
Thanks for the suggestion. I have added myself as the maintainer. 

https://github.com/NixOS/nixpkgs/pull/261397/commits/6faa74b47849a3d161bf7747c3505c8185a82334
https://github.com/NixOS/nixpkgs/pull/261397/commits/b819b61d91b61789e3f59005ebb0645c07cf7e73#diff-fa2d5b5d9f78fd26b25385f9f0f04acd1f4b57d2f8cac71b96faa217dc47c2c1R42